spec/orm/activerecord_spec.rb

```diff
@@ -1,26 +1,26 @@
 require 'spec_helper'
 require 'support/activerecord'
 
-def create_table(name)
-  ActiveRecord::Base.connection.create_table(name, force: true) do |t|
-    t.column :image, :string
-    t.column :images, :json
-    t.column :textfile, :string
-    t.column :textfiles, :json
-    t.column :foo, :string
+describe CarrierWave::ActiveRecord, :aggregate_failures do
```

Comment:
Could you remove this `:aggregate_failures` ? While it can be useful for local development, I don't see any benefit for permanently enabling this.

Reply:
I could remove it, but there's no harm in keeping it, as it's also useful in CI. It gives better errors when things fail, and has no overhead in the success case.